core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/Purchases.android.kt

```diff
@@ -185,15 +187,18 @@ public actual class Purchases private constructor(private val androidPurchases:
         onSuccess: (storeTransaction: StoreTransaction, customerInfo: CustomerInfo) -> Unit,
         isPersonalizedPrice: Boolean?,
         oldProductId: String?,
-        replacementMode: GoogleReplacementMode,
+        replacementMode: ReplacementMode?,
```

Comment:
Note this is an additional change I made. I feel it has more sense. I also wonder if we should move to the builder pattern for the purchase methods, as we do in `purchases-android`... But this works for now.